src/main/java/de/orchound/gameoflife/parsing/RleBoardParser.java

```diff
@@ -0,0 +1,60 @@
+package de.orchound.gameoflife.parsing;
+
+import de.orchound.gameoflife.game.Board;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class RleBoardParser implements BoardParser {
+
+	private Board board;
+
+	@Override
+	public Board parse(File file) {
+		try {
+			Files.lines(file.toPath()).filter(line -> !line.startsWith("#")).forEach(this::parseLine);
+		} catch (IOException e) {
+			e.printStackTrace();
+		}
+		return board;
+	}
+
+	private void parseLine(String line) {
+		Pattern dimensionPattern = Pattern.compile("x\\s?=\\s?(\\d+),\\s?y\\s=\\s?(\\d+)");
```

Comment:
You are compiling the pattern for every line your are parsing.
Promote the pattern to become a class member.